src/components/table/examples/table-layout-fitColumns.tsx

```diff
@@ -0,0 +1,44 @@
+import { Component, h, State } from '@stencil/core';
+import { Column } from '../table.types';
+import { invoices, Invoice } from './invoices';
+
+/**
+ *
+ * ````tsx
+ * layout="fitColumns"
+ * ````
+ *
+ * With this layout, the table stretches columns so that all
```

Comment:
A similar (to layout="fitDataStretch") problem appears if you resize the columns  

https://user-images.githubusercontent.com/50618208/188550499-c1d839ef-08a2-48c2-809f-5f7bcad4118d.mov

Reply:
Good finding! I didn't notice this. But this is what we get from Tabulator and cannot do anything about it.

I guess we can make resizability as a customizable prop

Reply:
The last column works fine in all other cases except these two `limel-example-table-layout-fit-data-stretch`  `limel-example-table-layout-fit-columns`
I like the column behavior in `limel-example-table-layout-fit-data` 👇
https://user-images.githubusercontent.com/50618208/188583001-7884a90d-4fab-4e3a-b077-e3f7be6d635e.mov

Would it be possible to have the same approach for that two?

Reply:
1. While column resizability being optional may perhaps be a feature we want anyway, I don't think we should consider something like that to be a fix for a bug. Taking user adjustability away just to force something to line up—not just before the user changes anything, but always—is a bad design choice, in my opinion.
2. Unless I'm missing something here, I don't think it's a bug that the columns don't fill the whole width _after_ the user changes the width of one or more columns. Quite the opposite, I find it very annoying when a set of columns are locked to a certain width, so I can't change the width of one column without some other column also changing to "cancel out" the width difference 😤
It should be enough that the columns spread to take the full width by default. If the user wants to change that, let them change it. Power to the people! ✊

Reply:
I agree with both of you and your preferences. As I said, the way these columns are rendered is coming from Tabulator. It's not my choice. Otherwise in most cases I'd also prefer to visually stretch out the row to the end of the table (just like the default). So if you resize the table columns manually, you don't get empty space on the right side.

Anyhow, this is what they give us. We're now only enabling the features that they have for our consumers.

## resizability of columns
If I don't get this wrong, this is already an optional feature available on individual columns: http://tabulator.info/docs/5.3/layout#resize-column we're just not documenting it here in the Docs, and don't have examples for it. I haven't tested it though, but I assume it works.

However, the consumer/designer can and should have this choice. Sometimes, for instance when there are very few columns and one of these layouts has been chosen. In some scenarios, the design choice might be no letting the end-user manipulate the column widths. Specially if the layout is "fitting the intended need". However, the consumer can use this feature in the wrong way, like any other feature. And we can't do anything about it.